NOTES-Perl.md

```diff
@@ -55,14 +55,14 @@
  ourselves to core Perl modules to keep the requirements down.  There
  are just a few exceptions:
 
- Test::More         We require the minimum version to be 0.96, which
+    Test::More      We require the minimum version to be 0.96, which
```

Comment:
It's unfortunate that markdown doesn't have a recognised syntax for definition lists, that would be suitable here.

That being said, I think it would be preferable to make a proper list of this, so something like this:

``` md
 * Test::More

   We require the minimum version to be 0.96, which appeared in Perl 5.13.4,
   because that version was the first to have all the features we're using.
   This module is required for testing only!  If you don't plan on running the tests,
   you don't need to bother with this one.

 * Text::Template

   This module is not part of the core Perl modules.  As a matter of fact, the core
   Perl modules do not include any templating module to date.  This module is
   absolutely needed, configuration depends on it.
```

Reply:
I wasn't overly careful with the line length, please adapt appropriately

Reply:
> It's unfortunate that markdown doesn't have a recognised syntax for definition lists, that would be suitable here.

Indeed.

> That being said, I think it would be preferable to make a proper list of this,

Done.